webui/react/src/pages/TrialDetails/TrialRangeHyperparameters.module.scss

```diff
@@ -92,11 +92,12 @@
 
   &::after {
     background-color: white;
+    border-radius: 2px;
     content: '';
-    height: 25px;
-    left: 8px;
+    height: 100%;
+    left: 8.5px;
```

Comment:
suggestion: could some of these values be arrived at through sass [variable arithmetics](https://sass-lang.com/documentation/operators/numeric) . it'd be only valuable if it makes it easier to understand how they relate to eachother